src/week1.org

```diff
@@ -0,0 +1,124 @@
+#+TITLE: Week 1
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org
+
+* What are we doing today?
+- Trainer introductions
+- Overview of the RoboJackets software training program
+- Navigating and manipulating files and directories in the Linux terminal
+- Using Git and GitHub for version control
+
+* Who am I? - Kristaps
```

Comment:
Unfortunately, the slides getting cut off is a longstanding problem in `reveal.js`, and I don't think they're planning on getting it fixed any time soon. AFAIK it's something about how it's hard to scale things by their height in CSS.

At any rate, the best solution I've found is to add a scrollbar to slides that overflow. 
https://stackoverflow.com/questions/35237862/best-way-to-handle-overflowing-reveal-js-slide

I can go ahead and add that, but if anyone has better CSS knowlege than me, if you come up with a diff or a css class that resizes the slides properly, I'll find a way to apply it to the slides.

That said, I think that if these slides are scaled down at all, the images would be hard to see (they're pretty hard to see right now) on a projector.

I would reccomend making the images into either vertical child slides of the about slides, or background images.  

Lastly, If anyone thinks upgrading our reveal version will help, let me know and I'll go ahead and do that. We're using 3.0.0, while 3.5.0 has been released. 

For more information, you might want to look at the [presentation size docs](https://github.com/hakimel/reveal.js#presentation-size). Right now I have the margins set to 0.22.

Reveal js targets a specific presentation size, and resizes based on that. If you overflow the presentation size, your SOL. 
![2017-07-04-100647_1411x975_scrot](https://user-images.githubusercontent.com/4349709/27838660-fc6301dc-60a0-11e7-8ed0-b00b55115439.png)

FWIW, if we decide to target 1080p, all the slides fit fine, since there's more than enough space, but projectors might not be 1080p, so that's proably a bad idea.